MMA.md

```diff
@@ -0,0 +1,365 @@
+  "timezone" -> "Asia/Taipei"           // System Timezone.
+  "versionName" -> "55.0a1"             // Fennec version.
+  "systemVersion" -> "10.3.1"            // System version.
+  "deviceModel" -> "iPhone"             // System device model.
+  "timezoneOffsetSeconds" -> "28800"    // User timezone offset with PST.
+  "deviceName" -> "sdaswani-31710"       // System device name.
+  "region" -> "(detect)"                // Not used. We strip location so this is will be the default stub value in Leanplum SDK.
+  "city" -> "(detect)"                  // Same as above.
+  "country" -> "(detect)"               // Same as above.
+  "location" -> "(detect)"              // Same as above.
+  "newsfeedMessages" -> " size = 0"     // Not used. New Leanplum Inbox message(Leanplum feature) count.
+  "includeDefaults" -> "false"          // Not used. Always false.
+
+  // Other life cycle actions
+  "action" -> "heartbeat"               // heartbeat: every 15 minutes when app is in the foreground
+                                        // pauseSession: when app goes to background
+                                        // resumeSession: when app goes to foreground
+
+  // Sent for every action
+  "userId" -> "b13b3c239d01aa7c"        // Set by Fennec, we use random uuid so users are anonymous to Leanplum.
+  "deviceId" -> "b13b3c239d01aa7c"      // Same as above.
+  "sdkVersion" -> "2.2.2-SNAPSHOT"      // Leanplum SDK version.
+  "devMode" -> "true"                   // If the SDK is in developer mode. For official builds, it's false.
+  "time" -> "1.497595093902E9"          // System time in second.
+  "token" -> "nksZ5pa0R5iegC7wj...."    // Token come from Leanplum backend.
+
+~~~~~~~~~~~~~~~
+
+Notes on what data is collected
+-------------------------------
+
+User Identifier:
+Since Device ID is a random UUID, Leanplum can't map the device to any know Client ID in Fennec nor Advertising ID.
+
+Events:
+Most of the Leanplum events can be mapped to a single combination of Telemetry event (Event+Method+Extra).
+Some events are not collected in Mozilla Telemetry. This will be addressed separately in each campaign review.
+There are three elements that are used for each event. They are: event name, value(default: 0.0), and info(default: "").
+Default value for event value is 0.0. Default value for event info is empty string.
+
+Here is the list of current Events sent, which can be found here in the code base: https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Application/LeanplumIntegration.swift#L21
+
+The first launch after install
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_First_Run",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
```

Comment:
Let's not make this too complicated - the only thing that differs between each event here is the event name, so that's sufficient to put here, which will make it easier for future changes to be added (in that you'll just need to add the event name, not the full ping).

I am a tiny bit confused here, but can you confirm: the fields listed in this ping are already covered in the section above, the part prefaced by "sends the following information about the start of a new session with the Leanplum backend", is that correct? There are some differences (e.g. this ping is less than what is described above) but if the above-parameters are correct, then we can just document those above, and not need the duplication here. Why are some fields missing here, but included in the section above? If they're not actually being sent, can we just remove those fields from the section above?